core/src/main/java/org/apache/iceberg/PartitionData.java

```diff
@@ -96,6 +96,7 @@ public Schema getSchema() {
     return schema;
   }
 
+  @Override
   public Type getType(int pos) {
     return partitionType.fields().get(pos).type();
   }
```

Comment:
I want to get PartitionData field type, I don't know how to get it in other way.